forge/licensing/index.js

```diff
@@ -64,6 +64,9 @@ module.exports = fp(async function (app, opts, next) {
             }
             return defaultLimits[key]
         },
+        get status () {
```

Comment:
As status is computed and not a static property, this should be a function not a getter for consistency.

Reply:
resolved in [9900fb7](https://github.com/flowforge/flowforge/pull/1757/commits/9900fb7da57ea58df5c7333193508fb1e68badf2)